app/assets/javascripts/organisation-list-filter.js

```diff
@@ -75,6 +81,12 @@
       });
 
       $('.js-no-filter-matches').toggleClass('js-hidden', totalMatchingOrgs > 0);
+
+      var text = ' results found'
```

Comment:
You could one-line this declaration followed by an if statement. I'm open to if this makes the line more difficult to read. Ideally we'd have ES6 template literals which would make this dreamy to read.

```suggestion
      var text = ' result' + totalMatchingOrgs === 1 ? '' : 's' +  ' found'
```

Reply:
Definitely more efficient, definitely harder to read 😁

Reply:
Point taken. Comment withdrawn.